sites/standalone-swarm.conf

```diff
@@ -12,50 +12,33 @@ server {
     include /etc/nginx/ssl.default.conf;
 
     server_name
-        aggregation-caesar.zooniverse.org
-        caesar.zooniverse.org
         citizensciencetoday.org www.citizensciencetoday.org
         classic.oldweather.org
         contact.zooniverse.org
-        education-api-staging.zooniverse.org
         education-api.zooniverse.org
         faas.zooniverse.org
         firebase-token-generator.zooniverse.org
-        fe-project.preview.zooniverse.org
         forum.moonzoo.org
         forum.oldweather.org
         forum.solarstormwatch.com
         geordi.zooniverse.org
-        higgs-hunters-facebook.zooniverse.org
         hubble.galaxyzoo.org
         mico-demo.snapshotserengeti.org
         mechanical-staging.zooniverse.org
         mechanical.zooniverse.org
-        interventions-gateway-staging.zooniverse.org
-        interventions-gateway.zooniverse.org
         old.oldweather.org
         old.planethunters.org
         oldtalk.planethunters.org
-        passbolt.zooniverse.org
-        penguin-watch-fb.zooniverse.org penguin-watch-facebook.zooniverse.org
         portainer.zooniverse.org
-        ses-bounces.zooniverse.org
         talk.ancientlives.org
         talk.andromedaproject.org
         talk.batdetective.org
         talk.seafloorexplorer.org
         talk.setilive.org
-        thumbnails.zooniverse.org
-        visualize.galaxyzoo.org
-        whaling.oldweather.org
-        wildcam-gorongosa-facebook.zooniverse.org
         www.galaxyzooforum.org galaxyzooforum.org
-        www.measuringtheanzacs.org
         www.setilive.org
-        www.zooteach.org
         zoo3.galaxyzoo.org
         grafana-stats.zooniverse.org
```

Comment:
This has to move over with graphql stats

Reply:
Just to clarify, do I need to make any changes to the config as it is now? Or is this just a note for the future?

Reply:
If it can move now, please do. It's part of the same stack as graphql stats, etc though i only tigged that this may not have persistent storage right now.. up to you.

Reply:
Grafana has persistent storage, so I'm not moving `zoo-event-stats-graphql` just yet.